### ISSUE TITLE
Default toast status colors are not WCAG compliant

### ISSUE THREAD

**Issue by dpayk** [chakra-ui/chakra-ui#8002]
### Description

The toast component shows a toast in different colors when using the [status property](https://chakra-ui.com/docs/components/toast#status).

To reach a better accessibility, I suggest to adjust the colors to slightly darker variants.

### Problem Statement/Justification

The background colors together with the white text color result in a contrast below 4.5:1 which means it does not meet the minimum AA level for color contrast as [specified by the WCAG](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).

### Proposed Solution or API

This website for example provides a check for contrast: https://webaim.org/resources/contrastchecker/
